tests/test_util.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+from numpy.testing import assert_allclose
+import pytest
+import sfs
+
+test_data = [
```

Comment:
Since we might add more tests for ùtil` in the future the name of the array should be chose more specific.

Reply:
Noted! I will change it to reflect the test it is referenced in.